slave/executors.py

```diff
@@ -43,6 +43,7 @@ def execute(self, benchmarkDir, benchmark, args, env, config):
 
 class BrowserExecutor(object):
     def __init__(self, engineInfo):
+        self.logger = logging.getLogger('BrowserExecutor')
```

Comment:
light suggestion: Instead of having one that gets redefined by the child classes, maybe pass a `name` parameter to the ctor provided by the child classes, so the child classes don't need to redefine their own one?